test/integration_test/basic_test.go

```diff
@@ -243,3 +270,79 @@ func testInstallWithTelemetry(t *testing.T, cluster *corev1.StorageCluster) {
 	err = coreops.Instance().DeleteSecret(secret.Name, secret.Namespace)
 	require.NoError(t, err)
 }
+
+// BasicCsiRegression test includes the following steps:
+// 1. Deploy PX with CSI enabled and validate CSI components and images
+// 2. Delete "px-csi-ext" pods and validate they get re-deployed
+// 3. Disable CSI and validate CSI components got successfully removed
+// 4. Enabled CSI and validate CSI components and images
+// 5. Delete "portworx" pods and validate they get re-deployed
+// 6. Delete "px-csi-ext" pods and validate they get re-deployed
+// 7. Delete StorageCluster and validate it got successfully removed
+func BasicCsiRegression(tc *types.TestCase) func(*testing.T) {
+	return func(t *testing.T) {
+		if tc.ShouldSkip() {
+			t.Skip()
+		}
+
+		var err error
+		testSpec := tc.TestSpec(t)
+		cluster, ok := testSpec.(*corev1.StorageCluster)
+		require.True(t, ok)
+
+		// Create and validate StorageCluster
+		cluster = ci_utils.DeployAndValidateStorageCluster(cluster, ci_utils.PxSpecImages, t)
+
+		// Delete portworx oci-mon pods and validate they get re-deployed
```

Comment:
nit: Don't need to add the comment if we are already saying the same in the following comment.

Reply:
Remove this and other obvious comments